src/invoices/InvoiceDetails/VoucherInformation/VoucherInformation.js

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Col,
+  Headline,
+  Icon,
+  KeyValue,
+  MultiColumnList,
+  Row,
+} from '@folio/stripes/components';
+import { AmountWithCurrencyField } from '@folio/stripes-acq-components';
+
+import { formatDate } from '../../../common/utils';
+import {
+  VOUCHER,
+  VOUCHER_LINES,
+} from '../../../common/resources';
+
+const visibleColumns = ['externalAccountNumber', 'amount'];
+const columnMapping = {
+  externalAccountNumber: <FormattedMessage id="ui-invoice.invoice.details.voucher.externalAccountNumber" />,
+  amount: <FormattedMessage id="ui-invoice.invoice.details.voucher.amount" />,
+};
+const columnWidths = {
+  externalAccountNumber: '50%',
+  amount: '50%',
+};
+
+const VoucherInformation = ({ invoiceId, mutator, resources }) => {
```

Comment:
please make container that will load data

Reply:
@AndreiShumski2 
here is a good article with different component types https://medium.com/teamsubchannel/react-component-patterns-e7fb75be7bb0
cc @Feboll